x/programs/rust/wasmlanche_sdk/src/host/mod.rs

```diff
@@ -42,7 +42,12 @@ pub fn init_program_storage() -> ProgramContext {
     unsafe { ProgramContext::from(_init_program()) }
 }
 
-pub fn store_bytes(
+/// # Safety
+/// The caller must ensure that key_ptr + key_len and
+/// value_ptr + value_len point to valid memory locations.
+///
+/// Stores the bytes at value_ptr to the bytes at key ptr on the host.
```

Comment:
I believe the proper syntax is the safety notice goes below the docs for the method/fucntion

/// Docs
///
/// # Safety

ref. https://doc.rust-lang.org/src/core/ptr/const_ptr.rs.html#406

Reply:
```suggestion
/// Stores the bytes at value_ptr to the bytes at key ptr on the host.
///
/// # Safety
/// The caller must ensure that key_ptr + key_len and
/// value_ptr + value_len point to valid memory locations.
```